### ISSUE TITLE
Proofs are found for invalid inferences (APP under NOFACT)

### ISSUE THREAD

**Issue by assaft** [wenkokke/pelican#34]
Say that we have a pair:
T: John said that a body, which was found by the police, was buried.
H: A body was found by the police.

This inference is invalid if 'said' is interpreted and annotated as NOFACT. Yet, the system finds a proof for that.

Reason: the appositive in [[a body] [which was found by the police]] introduces an IOTA operator. This IOTA brings a presupposition that such a body exists. Therefore, the hypothesis follows. 

To avoid this incorrect inference, we need to block the presupposition projection from Appositives that are under the scope of NOFACT. Notice that if there was an explicit IOTA there, the presupposition should be projected nevertheless: 
T: John said that the tall body was buried.
H: Some body is tall.
Here the explicit IOTA in "the tall body" should allow the presupposition to project regardless of whether it's under the scope of FACT or NOFACT

So to sum up, the generalization is that presuppositions projected from appositives should be blocked by NOFACT. Presuppositions projected from explicit definite articles, shouldn't. I hope it is correct. Should be discussed further.

I categorized this as a bug because the situation is that annotators get proofs for data that are not properly annotated (a proof should be found only if annotated with FACT).
